web_development_101/project_html_css.md

```diff
@@ -1149,6 +1149,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 * [Akshat Dubey's Solution (Easy)](https://github.com/akshatdb/google-homepage) | [View in browser](https://akshatdb.github.io/google-homepage/)
 * [Siddharth Jaiswal's Solution(Easy)](https://github.com/Sidd0602/google-homepage/) | [View in browser](https://sidd0602.github.io/google-homepage/)
 * [Anik Pramanik's Solution(Easy)](https://github.com/radioactive001/google-homepage) | [View in browser](https://cdn.rawgit.com/radioactive001/google-homepage/0890d71e/index.html)
+* [KWarrick's Solution(Easy)](https://github.com/fullpruf/googlemock) | [View in browser}(https://github.com/fullpruf/googlemock)
```

Comment:
looks like you have a rouge curly brace here instead of a closing bracket `[View in browser}`